### PR TITLE
[ID] Add `Game modifier/Nightcore` and `Game modifier/ScoreV2`

### PR DESCRIPTION
**Pull request by wowcake**
Added the Indonesian translation for "Nightcore" and "ScoreV2" game modifier.

### CODE REVIEW

File: wiki/Game_modifier/Nightcore/id.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - NC
+  - mod
+  - game modifier
+---
+
+# Nightcore (mod)
+
+![Ikon mod NC](/wiki/shared/mods/NC.png "Ikon mod Nightcore (NC)")
+
+*Berbeda konteks dengan [Nightcore (genre)](https://en.wikipedia.org/wiki/Nightcore "Wikipedia").*\
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:14]:
```suggestion
*Berbeda konteks dengan [Nightcore (aliran)](https://en.wikipedia.org/wiki/Nightcore "Wikipedia").*\
```

Comment (mousesalt) [reply wiki/Game_modifier/Nightcore/id.md:14]: should we replace it instead, similar to the beatmap listing site?

File: wiki/Game_modifier/Nightcore/id.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Singkatan: NC
+- Tipe: Peningkat Kesulitan
+- Pengali Skor:
+  - ![][osu!]: 1.06x
+  - ![][osu!taiko]: 1.12x
+  - ![][osu!catch]: 1.06x
+  - ![][osu!mania]: 1.00x
+- Tombol shortcut default: `D` `D` atau `Shift`+`D`
+- Caption: `uguuuuuuuu`
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:29]:
```suggestion
- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/Nightcore/id.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Variasi: [Double Time](/wiki/Game_modifier/Double_Time)
+
+## Deskripsi
+
+Mod **Nightcore** adalah sebuah [game modifier](/wiki/Game_modifier) yang fungsinya sama dengan variasinya (Double Time), tetapi mod ini juga meningkatkan nadanya dan menambahkan sebuah suara drum di latar belakang yang — dalam waktu 4/4 — mengikuti urutan drum bass pada beat 1 dan 3, clap pada beat 2 dan 4, dan crash pada beat 1 setiap 4 kali.
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:34]:
```suggestion
Mod **Nightcore** adalah sebuah [game modifier](/wiki/Game_modifier) yang berfungsi sama seperti variasinya (Double Time), yang membedakan dengan variannya adalah mod ini meningkatkan nada dan menambahkan sebuah suara drum di latar belakang—dalam timing 4/4—dengan mengikuti urutan beat 1 dan 3 pada drum bass, beat 2 dan 4 pada clap, dan beat 1 setiap 4 measure pada crash.
```

File: wiki/Game_modifier/Nightcore/id.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+[Pengali skor](/wiki/Game_modifier/Score_multiplier) untuk mod Nightcore tetaplah sama seperti mod Double Time, dan beatmap tidak akan terpengaruhi selain dari pengaruh yang sudah ada pada mod Double Time.
+
+Mod ini mempunyai fungsi yang sama untuk semua mode-mode game.
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:38]:
```suggestion
Mod ini mempunyai fungsi yang sama untuk semua mode permainan.
```

File: wiki/Game_modifier/Nightcore/id.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Trivia
+
+- Mod Nightcore pada awalnya adalah sebuah [lelucon April Fools](https://osu.ppy.sh/community/forums/topics/49733) tapi kemudian ditambahkan kedalam permainan sebagai mod yang benar-benar bisa dimainkan.
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:42]:
```suggestion
- Mod Nightcore pada awalnya adalah sebuah [lelucon April Fools](https://osu.ppy.sh/community/forums/topics/49733) tapi kemudian ditambahkan ke dalam permainan sebagai mod yang benar-benar bisa dimainkan.
```

File: wiki/Game_modifier/Nightcore/id.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- Mod Nightcore mendapatkan namanya dari sebuah genre (atau bagi beberapa orang, genre-sampingan) "Nightcore" dimana lagu diedit atau dibuat untuk dipercepat sekitar 10-30% dan mempunyai nada suara vokal yang lebih tinggi. Meskipun keberlakuan dan sejarah tentang genre ini masih kontroversial.
```
Comment (mousesalt) [inline wiki/Game_modifier/Nightcore/id.md:43]:
```suggestion
- Mod Nightcore mendapatkan namanya dari sebuah genre (atau bagi beberapa orang, sub-genre) "Nightcore" di mana lagu diedit atau dibuat untuk dipercepat sekitar 10-30% dan mempunyai nada suara vokal yang lebih tinggi. Meskipun keberlakuan dan sejarah tentang genre ini masih kontroversial.
```

File: wiki/Game_modifier/ScoreV2/id.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [Score](/wiki/Score) atau [ScoreV1](/wiki/Score/ScoreV1).*
+
+## Tentang
+
+- Singkatan: SV2
+- Tipe: Spesial
+- Pengali Skor: 1.00x
+- Caption: `Cobalah sistem skoring untuk masa depan. ** UNRANKED **`
```
Comment (mousesalt) [inline wiki/Game_modifier/ScoreV2/id.md:22]:
```suggestion
- Caption: `Cobalah sistem skoring masa depan. ** UNRANKED **`
```

File: wiki/Game_modifier/ScoreV2/id.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```
Comment (mousesalt) [inline wiki/Game_modifier/ScoreV2/id.md:23]:
```suggestion
- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/ScoreV2/id.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+## Deskripsi
+
+*Peringatan: Menggunakan mod ScoreV2 akan membuat unranked play.*
+
+*Untuk informasi mengenai sistem skoring ScoreV2, kunjungi: [ScoreV2](/wiki/Score#scorev2)*
+
+Mod **ScoreV2** adalah sebuah mod yang menggunakan sistem nilai prototipe ScoreV2 daripada sistem skoring yang digunakan sekarang pada setiap [game mode](/wiki/Game_mode). Singkatnya, sistem ScoreV2 mengukur semua nilai skor [hit object](/wiki/Hit_object) ke batas total skor 1 juta pada pengali skor 1.00x (terkecuali nilai bonus spinner pada [osu!](/wiki/Game_mode/osu!), denden pada [osu!taiko](/wiki/Game_mode/osu!taiko), dan banana yang dikumpulkan pada [osu!catch](/wiki/Game_mode/osu!catch).
```
Comment (mousesalt) [inline wiki/Game_modifier/ScoreV2/id.md:31]:
```suggestion
Mod **ScoreV2** adalah sebuah [game modifier](/wiki/Game_modifier) yang menggunakan sistem nilai prototipe ScoreV2 sebagai gantinya dari sistem skoring yang digunakan sekarang pada setiap [mode permainan](/wiki/Game_mode). Singkatnya, sistem ScoreV2 mengukur semua nilai skor [hit object](/wiki/Hit_object) ke batas total skor 1 juta pada pengali skor 1.00x (dikecualikan untuk nilai bonus spinner pada [osu!](/wiki/Game_mode/osu!), denden pada [osu!taiko](/wiki/Game_mode/osu!taiko), dan banana yang dikumpulkan pada [osu!catch](/wiki/Game_mode/osu!catch).
```

File: wiki/Game_modifier/ScoreV2/id.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+Mod ini mempunyai fungsi yang sama untuk semua mode-mode game.
```
Comment (mousesalt) [inline wiki/Game_modifier/ScoreV2/id.md:33]:
```suggestion
Mod ini mempunyai fungsi yang sama untuk semua mode permainan.
```

File: wiki/Game_modifier/ScoreV2/id.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+## Trivia
+
+- Mod ScoreV2 diperkenalkan pada Februari 22 (2017-02-22) versi [b20170222.3 (Stable)](https://osu.ppy.sh/home/changelog/stable40/20170222.3)
```
Comment (mousesalt) [inline wiki/Game_modifier/ScoreV2/id.md:37]:
```suggestion
- Mod ScoreV2 diperkenalkan pada 22 Februari 2017 (22/02/2017) versi [b20170222.3 (Stable)](https://osu.ppy.sh/home/changelog/stable40/20170222.3)
```

File: wiki/Game_modifier/Nightcore/id.md (hunk 1 above)
Comment (wowcake) [reply wiki/Game_modifier/Nightcore/id.md:14]: i don't think so...

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**